CMakeModules/BuildErlang.cmake

```diff
@@ -34,17 +34,43 @@ macro(rebar3_packbeam project_name)
     foreach(dep ${rebar3_packbeam_${project_name}_deps})
         add_custom_target(
             ${project_name}_${dep}_checkouts ALL
-            WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+            WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
             COMMAND mkdir -p _checkouts && cd _checkouts && ln -f -s ${CMAKE_SOURCE_DIR}/libs/${dep}
             VERBATIM
         )
     endforeach()
 
+    add_custom_command(
+        OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/src
+        COMMAND ln -s -f ${CMAKE_CURRENT_SOURCE_DIR}/src
```

Comment:
We don't support it right now, but it would be nice (see #8) and this change would completely rule it out.

Reply:
Well, not unless we use something like `mklink`.  I suspect there are many other issues with Windows that would need to be addressed.